php/Terminus/Models/Workflow.php

```diff
@@ -81,6 +81,22 @@ public function isSuccessful() {
   }
 
   /**
+   * Detects if the workflow has Operation Logs
+   *
+   * @return [boolean] $has_logs True if worklow has logs
+   */
+  public function hasLogs() {
+    $operations = $this->operations();
+    $has_logs = false;
+    foreach ($operations as $operation) {
+      if ((boolean)$operation->get('log_output')) {
+        $has_logs = true;
+      }
```

Comment:
It's a poor-person's inject: it only sets the value to be true if one of the outputs has logs. I should just write it as an actual inject to _reduce_ confusion :-)